CodeReady Containers/DaemonCommander/Handlers.swift

```diff
@@ -184,29 +184,11 @@ func HandleStart() {
         let appDelegate = NSApplication.shared.delegate as? AppDelegate
         appDelegate?.initializeMenus(status: "Starting")
     }
-    var response: Data
     do {
-        response = try SendCommandToDaemon(HTTPMethod.GET, "/api/start")
-        displayNotification(title: "Starting Cluster", body: "Starting OpenShift Cluster, this could take a few minutes..")
+        _ = try SendCommandToDaemon(HTTPMethod.GET, "/api/start")
+        displayNotification(title: "Successfully started cluster", body: "OpenShift cluster is running")
```

Comment:
Maybe show the notification before sending the command to the daemon?

Reply:
done 👍